python/SECRETSMANAGER_MAX_SECRET_AGE/SECRETSMANAGER_MAX_SECRET_AGE.py

```diff
@@ -0,0 +1,424 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+import json
+import sys
+import datetime
+from datetime import datetime, timedelta, timezone
+import boto3
+import botocore
+
+try:
+    import liblogging
+except ImportError:
+    pass
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::SecretsManager::Secret'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+# Other parameters (no change needed)
+CONFIG_ROLE_TIMEOUT_SECONDS = 900
+
+#############
+# Main Code #
+#############
+
+
+def evaluate_secret_compliance(valid_rule_parameters, secret):
+    #now = datetime.now(timezone.utc)
+    delta = timedelta(days=int(valid_rule_parameters.get('max_secret_age_days') if valid_rule_parameters.get('max_secret_age_days') else '30'))
+    max_secret_age = NOW - delta
+
+    if secret.get('LastRotatedDate'):
+        if datetime.replace(secret.get('LastRotatedDate'), tzinfo=timezone.utc) > max_secret_age:
+            return 'COMPLIANT'
+        return 'NON_COMPLIANT'
+
+    secret_versions = AWS_SECRETSMANAGER_CLIENT.list_secret_version_ids(
```

Comment:
Can you please add a comment that we don't need a paginator, because this case should only be possible when a secret has never been rotated, and therefore it must have only 1 version.

Reply:
`    # Pagination of this API call is not needed as this API is only called if Secret has never been rotated
    # This should always return only a single VersionId with VersionLabel AWSCURRENT`